app/i18n/translations/original.json

```diff
@@ -549,7 +549,7 @@
   "snackbar.defaults.error": "{originalError}",
   "snackbar.defaults.success": "Success!",
   "spv.auto.buyer.warn": "Ticket Auto Buyer not available while using SPV",
-  "spv.purchase.warn": "Purchase Tickets is not available right now,\n  because we are in the beginning of a ticket interval.\n  after {blocksNumber} blocks it will be available again.",
+  "spv.purchase.warn": "Purchase Tickets is not available right now,\n  because we are in the beginning of a ticket interval.\n  After {blocksNumber} blocks it will be available again.",
```

Comment:
In general, there's no need to update the original.json file as well since this is also generated automatically before a release.

You can keep this for this particular PR though, since we're pretty close to final release.